source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with **GeoJSON**.
+GeoJSON is a geospatial data format created by the Internet Engineering
+Task Force (IETF). You can use the GeoJSON format to store your data in MongoDB,
+and you must use the GeoJSON format to query your geospatial data in MongoDB. 
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
+      ]
+   }
+
+For definitive information on GeoJSON, see the
+:rfc:`official IETF specification <7946>`.
+
+.. external resource
+
+GeoJSON Positions
+~~~~~~~~~~~~~~~~~
+
+A position is the base unit in GeoJSON, and represents a single place on Earth.
+A position is an array containing two or three number values: 
+
+- a longitude in the first position
+- a latitude in the second position
+- an optional elevation in the third position 
```

Comment:
suggestion: reduce repetition. Also, I think it's easy to misconstrue "single place" as "single point". Lastly, base unit doesn't strictly match the RFC, nor does it help the user work with GeoJSON in this article, I feel.

```suggestion
A position represents a location on Earth, and is comprised as 
an array containing two or three number values: 

- Longitude in the first position (required)
- Latitude in the second position (required)
- Elevation in the third position (optional)
```

Reply:
Thanks for the note. I agree that "base unit" is the incorrect term. The specification uses "fundamental geometry construct", which I agree is not the same thing as "base unit". I changed "is comprised of" to "exists in code as" to [avoid passive voice](https://dictionary.cambridge.org/us/grammar/british-grammar/consist-comprise-or-compose). 

Going forward, I will try to continue to refine my sense for when repetition is unhelpful.

I really like the edit you have for the list of array values! Having required/optional at the end is great, and removing a/an is great also.

Fix: https://github.com/mongodb/docs-java/pull/78/commits/28d9772c7463b4c18d3522eb444788775b0bde66